ynr/apps/people/templates/people/correct_not_standing_in_merge.html

```diff
@@ -0,0 +1,66 @@
+{% extends "base.html" %}
+{% block content %}
+
+  <h2>'Not standing' mix up</h2>
+  <p>Merging these profiles would cause the resulting profile to be
+    added to an election that someone has asserted the person isn't standing in.</p>
+  <p>This can sometimes happen when a duplicate profile has been created, but it could be
+  some other problem that needs investigating.</p>
+  <p>Please check that this merge is correct, and if you think everything looks ok you
+  can remove the "not standing" assertion and continue with the merge.</p>
```

Comment:
Minor point, but can we change the wording on this last sentence slightly so that the advice is phrased in terms of the text on the buttons e.g: 'if you think everything looks ok, choose "confirm" to remove the "not standing" assertion' or something.